apps/decoder_gtfs/lib/decoder/gtfs.ex

```diff
@@ -0,0 +1,40 @@
+defmodule Decoder.Gtfs do
```

Comment:
These 'wholly new' decoder modules and tests are actually just renames, right?

Reply:
yes